hack/conformance/check_conformance_test_requirements.go

```diff
@@ -56,7 +56,7 @@ func checkAllProviders(e2eFile string) error {
 		line := scanner.Text()
 		if regStartConformance.MatchString(line) {
 			if inConformanceCode {
-				return errors.New("Missed the end of previous conformance test. There might be a bug in this script.")
+				return errors.New("missed the end of previous conformance test. there might be a bug in this script")
```

Comment:
Thanks for your PR. Since the whole file is consistent with having the error messages start with Caps, that should be ok.

Reply:
I run golint ,it shows as :
<img width="1434" alt="image" src="https://user-images.githubusercontent.com/31728060/210569121-62828655-a379-4f4b-a42a-590fbfd1c31b.png">
Thanks